consensus/broadcast/metaChainMessenger_test.go

```diff
@@ -303,3 +306,95 @@ func TestMetaChainMessenger_BroadcastBlockDataLeader(t *testing.T) {
 		assert.Equal(t, len(transactions), numBroadcast)
 	})
 }
+
+func TestMetaChainMessenger_Close(t *testing.T) {
+	t.Parallel()
+	args := createDefaultMetaChainArgs()
+	closeCalled := false
+	delayedBroadcaster := &mock.DelayedBroadcasterMock{
+		CloseCalled: func() {
+			closeCalled = true
+		},
+	}
+	args.DelayedBroadcaster = delayedBroadcaster
+	mcm, _ := broadcast.NewMetaChainMessenger(args)
+	require.NotNil(t, mcm)
+	mcm.Close()
+	assert.True(t, closeCalled)
+}
+
+func TestMetaChainMessenger_PrepareBroadcastHeaderValidator(t *testing.T) {
+	t.Parallel()
+	t.Run("Nil header", func(t *testing.T) {
```

Comment:
```suggestion
	t.Parallel()
	
	t.Run("Nil header", func(t *testing.T) {
```
optional suggestion: to add more spacing between inner tests